### ISSUE TITLE
RA fullscreen thumbnail enable+disable issue (rgui)+bug

### ISSUE THREAD

**Issue by riggles1** [libretro/RetroArch#14904]
A version or two ago prior to 1.14.0 when pressing the button "Y" in a playlist it could show a fullscreen thumbnail.
now "Y" will only toggle between art/screenshot/title thumbnails to the side.

The big issue with this is that us CRT rgui users don't want the tiny (very low res) thumbnails on the side taking up unecessary space on the playlist and covering entry text. 

The only way to get a fullscreen screenshot now is if you have the tiny side of the screen thumbnails enabled as well, pressing start makes it fullscreen.

The wanted behavior is that when thumbnails are disabled, that pressing "Y" (or Start) would still show a fullscreen screenshot of the game like before.

The bugged aspect of it:
First after enabling thumbnails nothing happens, or you just one of the artworks, however after pressing "Start" (to see a fullscreen preview) then again to return, both artworks show no matter your Appearance settings for if art/screenshot/title should be on the top or bottom.

But my main issue is that these thumbnails won't go away if I want to have a fullscreen view to begin with.

https://i.imgur.com/q4xNy87.png

Appreciate any help with this!

In short to be clear: I usually just want clean simple lists with no thumbnails, but every now and then I needed a reminder on what the game was and would press "Y" in order to see a fullscreen screenshot, and "Y" again to return to the normal text-only list.
